core/src/test/scala/unit/kafka/server/FetchSessionTest.scala

```diff
@@ -761,16 +836,16 @@ class FetchSessionTest {
       request2.isFromFollower,
       request2.fetchData(topicNames),
       request2.forgottenTopics(topicNames),
-      topicIds
+      topicNames
     )
 
     assertEquals(classOf[SessionErrorContext], context2.getClass)
-    val respData2 = new util.LinkedHashMap[TopicPartition, FetchResponseData.PartitionData]
+    val respData2 = new util.LinkedHashMap[TopicIdPartition, FetchResponseData.PartitionData]
     assertEquals(Errors.FETCH_SESSION_TOPIC_ID_ERROR,
       context2.updateAndGenerateResponseData(respData2).error())
   }
 
-  @Test
+  /*@Test
```

Comment:
I commented out this test, since I removed the behavior to catch inconsistent IDs at this stage. I can try to simulate catching the inconsistent ID later, but not sure if that is helpful to show in a test.